src/test/AludelV3.t.sol

```diff
@@ -28,9 +28,9 @@ import {Utils} from "./Utils.sol";
 import {UserFactory} from "./UserFactory.sol";
 
 import "forge-std/console2.sol";
+import "forge-std/StdUtils.sol";
```

Comment:
I understand this concern for contracts that are deployed but I don't see the point of enforcing this in tests.
I would prefer to have the whole package available instead of having to specify each of the names I want to use.

Reply:
you can do `import "forge-std/StdUtils" as STD` and access every exported thing as `STD.thing`

this'd not pollute the global namespace, and would make explicit where every usage comes from. In that way, if I want to find the docs for a particular function when reviewing the code or taking a look at it some time later, I can easily figure out where to look, whereas if the name is in the global namespace I'd have to do a fair bit more guessing